docs/content.zh/docs/ops/metrics.md

```diff
@@ -28,26 +28,26 @@ under the License.
 
 # 指标
 
-Flink exposes a metric system that allows gathering and exposing metrics to external systems.
+Flink 公开了一个指标系统，允许收集和公开指标给外部系统。
 
-## Registering metrics
+## 注册指标
 
-You can access the metric system from any user function that extends [RichFunction]({{< ref "docs/dev/datastream/user_defined_functions" >}}#rich-functions) by calling `getRuntimeContext().getMetricGroup()`.
-This method returns a `MetricGroup` object on which you can create and register new metrics.
+你可以通过调用从 [RichFunction]({{< ref "docs/dev/datastream/user_defined_functions" >}}#rich-functions) 
+扩展的任何用户函数的 `getRuntimeContext().getMetricGroup()` 方法访问指标系统。
+此方法返回一个 `MetricGroup` 对象，你可以在该对象上创建和注册新指标。
 
-### Metric types
+## 指标类型
 
-Flink supports `Counters`, `Gauges`, `Histograms` and `Meters`.
+Flink 支持计数器 `Counters`，量表 `Gauges`，直方图 `Histogram` 和仪表 `Meters`。
```

Comment:
计数器，量表，直方图 和仪表 -> What about keeping original content?

Reply:
because I saw the `https://nightlies.apache.org/flink/flink-docs-master/zh/docs/dev/python/table/metrics/`.
![image](https://user-images.githubusercontent.com/33420316/144000585-7999f6e7-0a49-4eb2-bc31-ad6a5fe9c391.png)